zerver/tests/test_channel_fetch.py

```diff
@@ -663,13 +664,26 @@ def test_subscriber(self) -> None:
 
     def test_gather_partial_subscriptions(self) -> None:
         othello = self.example_user("othello")
+        idle_users = [
+            create_user(
+                email=f"original_user{i}@zulip.com",
+                password=None,
+                realm=othello.realm,
+                full_name=f"Full Name {i}",
+            )
+            for i in range(300)
```

Comment:
ouch, no, don't create 300 users in a loop, that will make this test **incredibly** slow. 

Instead of hardcoding 250, put it in `zproject/default_settings.py` as `MIN_PARTIAL_SUBSCRIBERS_CHANNEL_SIZE = 250` and then this test can do `@override_settings(MIN_PARTIAL_SUBSCRIBERS_CHANNEL_SIZE=2)`.

Reply:
Thanks, that makes sense! I'm going to do `@override_settings(MIN_PARTIAL_SUBSCRIBERS_CHANNEL_SIZE=5)` so that we can test above and below the threshold more easily.

Reply:
Great. Avoiding incredibly slow tests is really important, because it's very easy to fail to notice for years, just wasting everyone's time running the test suite.